doc/man/8/osdmaptool.rst

```diff
@@ -29,6 +29,11 @@ Options
    will simply make the tool print a plaintext dump of the map, after
    any modifications are made.
 
+.. option:: --dump <format>
+
+   displays the map in plain text when <format> is 'plain', 'json' if specified
```

Comment:
@nvedant07 NIT If this is a new sentence. How good to start with Capital letter?
My suggestion..
Your thoughts?

Reply:
@amitkumar50 I don't agree with this comment. The format used is consistent with the rest of the document.